start.py

```diff
@@ -908,3 +926,42 @@ def failure(timeout):
     wait_for_condition(condition=condition, condition_args=[deployment, cluster_name],
                        time_between_checks=3, timeout=600, time_to_success=30,
                        success=success, failure=failure)
+
+
+class Version:
```

Comment:
I wonder if this class is adding more complexity than it's worth. When I wrote it for SDC, it was because I wanted to treat special cases like `x.y.z-RC1` > `x.y.z-SNAPSHOT` and things like that, but since we only handle CDH releases, I think we may be okay just using `tuple` to convert version strings into tuples and then comparing them that way.

Reply:
Done.